cmd/serve/webdav/webdav.go

```diff
@@ -109,17 +113,25 @@ func newWebDAV(f fs.Fs, opt *httplib.Options) *WebDAV {
 		f:   f,
 		vfs: vfs.New(f, &vfsflags.Opt),
 	}
-
-	handler := &webdav.Handler{
+	webdavHandler := &webdav.Handler{
 		FileSystem: w,
 		LockSystem: webdav.NewMemLS(),
 		Logger:     w.logRequest, // FIXME
 	}
-
-	w.Server = httplib.NewServer(handler, opt)
+	w.webdavhandler = webdavHandler
+	w.httpserver = rhttp.NewServer(f, opt)
+	w.Server = httplib.NewServer(http.HandlerFunc(w.handler), opt)
 	return w
 }
 
+func (w *WebDAV) handler(rw http.ResponseWriter, r *http.Request) {
+	if (r.Method == "GET" || r.Method == "HEAD") && strings.HasSuffix(r.URL.Path, "/") {
```

Comment:
So this just serves the directories?

Is that enough to be useful?

Reply:
> So this just serves the directories?

Serving files is already handled by the WebDAV handler so I think the http handler only has to directories.